roles/pulp/templates/settings.py.j2

```diff
@@ -1,7 +1,21 @@
 CONTENT_ORIGIN="http://{{ ansible_hostname }}:8080"
+API_CONTENT_ORIGIN="http://{{ ansible_hostname }}:24817"
+CONTENT_SERVICE_ORIGIN="http://{{ ansible_hostname }}:24816"
```

Comment:
I don't think these settings exist. I only see `CONTENT_ORIGIN` (https://pulpproject.org/pulpcore/docs/admin/reference/settings/#content_origin) but not any of the others.